docs/options.md

```diff
@@ -20,6 +20,7 @@ argocd-diff-preview [FLAGS] [OPTIONS] --repo <repo> --target-branch <target-bran
 | Flag | Environment Variable | Default | Description |
 |------|---------------------|---------|-------------|
 | `--debug`, `-d` | `DEBUG` | `false` | Activate debug mode |
+| `--dry-run`, `-n` | `DRY_RUN` | `false` | Perform a trial run listing the applications that would be processed |
```

Comment:
```suggestion
```
This is a good addition! But since the release process for this tool is not very refined, I’ll update the docs later. If we merge it as is, the docs will be updated before a new version of the tool is released 😅